### PR TITLE
Remove releases table for larger screens

### PR DESCRIPTION
**Pull request by carkod**
## Done

Removed releases table [according to request](https://warthogs.atlassian.net/browse/WD-10408)

## QA

- Go to  https://ubuntu-com-13760.demos.haus/desktop/developers and check that the table below the "Security support
" graph is no longer showing. Compare with prod https://ubuntu.com/desktop/developers
- Test that on mobile it doesn't show https://ubuntu-com-13760.demos.haus/desktop/developers

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-10408

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)

### CODE REVIEW

File: templates/desktop/developers.html [hunk 1]
```diff
@@ -69,7 +70,9 @@ <h2>Security support <br aria-hidden/>for Ubuntu releases</h2>
   </div>
   <div class="u-fixed-width">
     <div class="u-hide--small" id="server-desktop-eol"></div>
-    {% include "about/release_cycles/releases-table.html" %}
+    <div class="u-hide--large u-hide--medium">
```
Comment (chillkang) [inline templates/desktop/developers.html:73]: @carkod It's supposed to show on the small screen, right?

Comment (carkod) [reply templates/desktop/developers.html:73]: Yes. The graph will become a table in small screens (that functionality was already there) and the table that we are huding here should show

Comment (chillkang) [reply templates/desktop/developers.html:73]: aha right, it's just that the QA process says that the mobile doesn't show the table so I got confused!

**Review by chillkang — APPROVED** [review]
LGTM👍

File: templates/desktop/developers.html (hunk 1 above)
Comment (carkod) [reply templates/desktop/developers.html:73]: Sorry my bad, I meant the complete opposite hah 😅

Comment (chillkang) [reply templates/desktop/developers.html:73]: haha that's what I thought :) no worries at all

**Pull request merged**